src/YioMap.js

```diff
@@ -69,30 +94,95 @@ export class YioMap extends LitElement {
     return this.#contentMap;
   }
 
+  set editLayer(value) {
+    const oldValue = this.#editLayer;
+    this.#editLayer = value;
+    if (!this.#userEditInteraction) {
+      return;
+    }
+    if (!value) {
+      if (
+        oldValue &&
+        this.#userEditInteraction.getEditLayer().getSource().getFeatures()
```

Comment:
You‘re still using deeply nested property acces here. This check could be done in the interaction‘s `clearEditSource()` method when it is modified to return `true` when there was something to clear, and `false` otherwise.